__tests__/apiTests.test.js

```diff
@@ -314,4 +315,34 @@ describe("GET /api/users",()=>{
                 })
             })
     })
+})
+describe("GET /api/articles?topicFilter",()=>{
+    test("get status code 200, and return all the articles with the specific topic filter",()=>{
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+            .then(({body})=>{
+                const {articles} = body
+                expect(articles.length).toBeGreaterThan(0)
```

Comment:
We could be more specific here as we know how many articles in our test data have the topic of Mitch. It would be good to ensure we have got the right quantity back.